src/Serilog.Settings.Configuration/Serilog.Settings.Configuration.csproj

```diff
@@ -5,7 +5,7 @@
     <VersionPrefix>3.3.1</VersionPrefix>
     <LangVersion>latest</LangVersion>
     <Authors>Serilog Contributors</Authors>
-    <TargetFrameworks>netstandard2.0;net451;net461</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net462</TargetFrameworks>
```

Comment:
Since 6.0 is LTS and brings in fewer dependencies, it might make sense to target it specifically here.

Reply:
> brings in fewer dependencies

What do you mean? After this pull request there are only 3 dependencies, all included unconditionally: `Microsoft.Extensions.Configuration.Binder`, `Microsoft.Extensions.DependencyModel` and `Serilog`.

I see many package authors add `net6.0` to the target frameworks and I don't understand why. I see why you'd target .NET 6 if you are making use of new features introduced in .NET 6 such as the new `DateOnly` and `TimeOnly` types. If there's no `#if NET6_0_OR_GREATER` in the code, I don't see the point of targeting .NET 6, but maybe I'm missing something and I'd be happy to be enlightened.

Reply:
You might be right; I'll need to dig back into it :-)

Last time I checked, I think the NuGet dependency resolution algorithm would mean that a package targeting netstandard2.0 might drag in more transitive dependencies than a package targeting the later TFM, but it's a hazy memory and might not matter here.

Let's leave it out, for now 👍

Reply:
I think what you are referring to was the situation before .NET Standard 2.0 where many `System.*` packages were brought in. This issue has been solved with .NET Standard 2.0 so indeed it doesn't matter here. There's a good discussion about that on Stack Overflow: [Why does my .NET Standard NuGet package trigger so many dependencies?](https://stackoverflow.com/questions/47365136/why-does-my-net-standard-nuget-package-trigger-so-many-dependencies/47366401#47366401)

Reply:
Thanks for the pointer; no, I remember the switch away from NETStandard.Library, my memory is of someone coming along after that and demonstrating to me why netstandard2.0 still wasn't the optimal TFM ... 🤔 I'll try to find that thread when I have some time :-)

Recently I've been enabling 5 or 6 when turning on non-null reference type checking, which avoids bundling types from the Nullable package on platforms that already carry them. Not relevant to the change here, though.

Reply:
Interesting post about  .NET Standard 2.0 vs .NET 6 targeting on Stack Overflow for the record: [Are there benefits in producing a .NET 6.0 version of a .NET Standard 2.0 library?](https://stackoverflow.com/questions/70778797/are-there-benefits-in-producing-a-net-6-0-version-of-a-net-standard-2-0-librar/72266562#72266562)